app/src/main/java/com/io/moviO/moviesList/MoviesListViewModel.kt

```diff
@@ -5,7 +5,7 @@ import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.io.moviO.data.DataResult
-import com.io.moviO.data.Movie
+import com.io.moviO.data.modelMovie.Movie
```

Comment:
modelMovie is a weird name for a package. Why not just movie? For data you could mimic the path in the url:

{base_url}/movie/ being the foundation for your package